zerocopy-derive/src/lib.rs

```diff
@@ -634,6 +635,47 @@ fn derive_from_zeros_struct(ast: &DeriveInput, strct: &DataStruct) -> proc_macro
 // - all of its variants are fieldless
 // - one of the variants has a discriminant of `0`
 
+fn find_zero_variant<'a>(variants: impl IntoIterator<Item = &'a Variant>) -> Option<usize> {
+    let mut current_implicit_discriminant = Some(0isize);
```

Comment:
Maybe we should call this `next_implicit_discriminant` since it describes the descriminant of the next variant we're about to process?

Reply:
Done